charts/mothership/values.yaml

```
@@ -21,7 +21,7 @@ node:
   count: 1
   loadBalancerExternal: false
   opGeth:
-    image: &op-geth-image stackupwallet/op-geth:v1.101315.2
+    image: &op-geth-image ghcr.io/planetarium/op-geth:latest
```

```suggestion
    image: &op-geth-image ghcr.io/planetarium/op-geth:v1.101315.2
```
I've just tagged it
https://github.com/planetarium/op-geth/pkgs/container/op-geth/238438298?tag=v1.101315.2